helm/reana/templates/reana-config.yaml

```diff
@@ -34,3 +34,9 @@ data:
     {{- end }}
     quota_enabled: {{ .Values.quota.enabled }}
     file_preview_size_limit: {{ int .Values.components.reana_ui.file_preview_size_limit }}
+    launcher_repositories:
+    {{- range .Values.ui_launcher_repositories | default (list) }}
+    - name: {{ .name | quote }}
+      url: {{ .url | quote }}
+      image_url: {{ .image_url | quote }}
+    {{- end }}
```

Comment:
I didn't know about it, nice! The "problem" with `toYaml` however is that as far as I have understood and tried, everything has to be already properly indented and quoted in the values file, so I guess it gives less flexibility to the admins. Since this whole `range` thing is not too big and gives the details about the properties in the list, I think we can keep it this way. Do you agree?